src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -1402,7 +1402,7 @@ public void deleteNode(UUID studyUuid, UUID nodeId, boolean deleteChildren, Stri
         startTime.set(System.nanoTime());
         DeleteNodeInfos deleteNodeInfos = new DeleteNodeInfos();
         deleteNodeInfos.setNetworkUuid(networkStoreService.doGetNetworkUuid(studyUuid));
-        boolean invalidateChildrenBuild = !EMPTY_ARRAY.equals(networkModificationTreeService.getNetworkModifications(nodeId));
+        boolean invalidateChildrenBuild = !deleteChildren && !EMPTY_ARRAY.equals(networkModificationTreeService.getNetworkModifications(nodeId));
```

Comment:
Fixes an unrelated bug : if a node's children are also deleted it is unnecessary to invalidate them